### ISSUE TITLE
librhash.dll: trying to print a message digest, which was not given at class initialization, should not return something that looks like a correct result

### ISSUE THREAD

**Issue by todeskurve** [rhash/RHash#267]
i use the python bindings which led me to this interesting case:
```
import rhash
hasher = rhash.RHash(rhash.SHA1, rhash.MD5)
hasher.update("CRC32 will probably be wrong")
hasher.finish()
print(hasher.hex(rhash.MD5))
print(hasher.hex(rhash.CRC32))
print(hasher.hex(rhash.SHA1))
```
this returns different but good looking CRC32 every time you run.

**Comment by todeskurve** [comment]
imho, this should be fixed in librhash though or am i using it wrong?

**Comment by rhash** [comment]
This issue has been fixed by d7bc9fa and 0d199f6 by another method. The downside is that the Python part will only work when using librhash compiled with the fix.

**Comment by todeskurve** [comment]
so when can we expect v1.4.5 to be available at sourceforge?

**Comment by rhash** [comment]
The fix included into [RHash v1.4.5](https://github.com/rhash/RHash/releases/tag/v1.4.5).
